tests/data/metadata_invalid.json

```diff
@@ -1 +1 @@
-{"error": {"format": {"Error:  Duplicate CellID(s) in metadata file": ["BM01_16dpp_AAGCAGTGGTAT"]}, "type": {"ERROR: value provided does not match convention type delaration for organism_age": [["BM01_16dpp_CGGTAAACCATT", "ValueError(\"could not convert string to float: 'foo'\",)"]]}, "convention": {"'ethnicity' is a dependency of 'ethnicity-ontology_label'": ["BM01_16dpp_AAGCAGTGGTAT", "BM01_16dpp_TAAGCAGTGGTA", "BM01_16dpp_CTAAGCAGTGGT", "BM01_16dpp_CGGTAAACCATT", "BM01_16dpp_AAGCAGTGGTAT"], "'sex' is a required property": ["BM01_16dpp_AAGCAGTGGTAT", "BM01_16dpp_TAAGCAGTGGTA", "BM01_16dpp_CTAAGCAGTGGT", "BM01_16dpp_CGGTAAACCATT", "BM01_16dpp_AAGCAGTGGTAT"], "None is not of type 'string'": ["BM01_16dpp_TAAGCAGTGGTA"], "'direct from donr - fresh' is not one of ['cell line', 'organoid', 'direct from donor - fresh', 'direct from donor - frozen', 'cultured primary cells']": ["BM01_16dpp_CTAAGCAGTGGT"], "'foo' is not of type 'number'": ["BM01_16dpp_CGGTAAACCATT"]}}, "warn": {"ontology": {"Warning: no ontology label supplied in metdata file for \"CL_0000066\" - no cross-check for data entry error possible": null, "Warning: no ontology label supplied in metdata file for \"NCBITaxon_9606\" - no cross-check for data entry error possible": null, "Warning: no ontology label supplied in metdata file for \"GAZ_00003181\" - no cross-check for data entry error possible": null, "Warning: no ontology label supplied in metdata file for \"EFO_0008919\" - no cross-check for data entry error possible": null, "Warning: no ontology label supplied in metdata file for \"UBERON_0001913\" - no cross-check for data entry error possible": null, "Warning: no ontology label supplied in metdata file for \"UO_0000036\" - no cross-check for data entry error possible": null}}}
\ No newline at end of file
+{"error": {"format": {"Error:  Duplicate CellID(s) in metadata file": ["BM01_16dpp_AAGCAGTGGTAT"]}, "type": {"ERROR: value provided does not match convention type declaration for organism_age": ["BM01_16dpp_CGGTAAACCATT"]}, "convention": {"'ethnicity' is a dependency of 'ethnicity__ontology_label'": ["BM01_16dpp_AAGCAGTGGTAT", "BM01_16dpp_TAAGCAGTGGTA", "BM01_16dpp_CTAAGCAGTGGT", "BM01_16dpp_CGGTAAACCATT", "BM01_16dpp_AAGCAGTGGTAT"], "'sex' is a required property": ["BM01_16dpp_AAGCAGTGGTAT", "BM01_16dpp_TAAGCAGTGGTA", "BM01_16dpp_CTAAGCAGTGGT", "BM01_16dpp_CGGTAAACCATT", "BM01_16dpp_AAGCAGTGGTAT"], "None is not one of ['yes', 'no', 'unknown']": ["BM01_16dpp_TAAGCAGTGGTA"], "None is not of type 'string'": ["BM01_16dpp_TAAGCAGTGGTA"], "'direct from donr - fresh' is not one of ['cell line', 'organoid', 'direct from donor - fresh', 'direct from donor - frozen', 'cultured primary cells']": ["BM01_16dpp_CTAAGCAGTGGT"], "'foo' is not of type 'number'": ["BM01_16dpp_CGGTAAACCATT"]}}, "warn": {"ontology": {"Warning: no ontology label supplied in metdata file for \"UO_0000036\" - no cross-check for data entry error possible": null}}}
```

Comment:
In "Warning: no ontology label supplied in metdata file for", the term "metdata" should be "metadata".

It looks like the proper place to fix that would be upstream, which would then be used to generate new test data with the typo fix.

Reply:
updated code and reference issues file in `4023a08`